journal-of-mammalogy.csl

```diff
@@ -140,9 +140,11 @@
       </if>
       <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference  song" match="any">
         <group prefix=". " delimiter=" ">
-          <label variable="page" form="short" text-case="capitalize-first" suffix=" "/>
-          <text variable="page"/>
-          <text term="in"/>
+          <group delimiter=" ">
+            <label variable="locator" form="short" text-case="capitalize-first"/>
```

Comment:
no -- in spite of the name of the macro, (feel free to change), this is called in the bibliography and thus was correct as page (though should be wrapped in a group as you do)